src/metadata/metadata_raw.c

```diff
@@ -295,6 +312,8 @@ static void _raw_ram_flush_all_complete(ocf_cache_t cache,
  * RAM Implementation - Flush all elements
  */
 static void _raw_ram_flush_all(ocf_cache_t cache, struct ocf_metadata_raw *raw,
+		ocf_flush_page_synch_t lock_page_pfn,
```

Comment:
These callbacks can be part of `struct ocf_metadata_raw`, as in practice they are bound to raw container context rather than to caller context.

Reply:
done